src/components/index.ts

```diff
@@ -31,7 +31,7 @@ export {
 
 export * from './navigationLists';
 export * from './inputs';
-export * from './modals/bottomModals';
-export * from './modals/centerModals';
-export * from './modals/ExploreModals';
+export * from './Modals/bottomModals';
+export * from './Modals/centerModals';
```

Comment:
better not to change filename, keep it as [modals] in everywhere and try "git config core.ignorecase false"
and make sure filename is changed in your branch also